.circleci/config.yml

```diff
@@ -857,6 +861,11 @@ jobs:
                 name: Install CocoaPods dependencies
                 command: |
                   rm -rf packages/rn-tester/Pods
+
+                  if [[ << parameters.architecture >> == "NewArch" ]]; then
+                    export RCT_NEW_ARCH_ENABLED=1
+                  fi
+
                   cd packages/rn-tester && bundle exec pod install
 
       - run:
```

Comment:
Could you also update the name of the task below by adding the architecture? An example of doing that is [here](https://github.com/facebook/react-native/blob/main/.circleci/config.yml#L790)

Reply:
> Could you also update the name of the task below by adding the architecture? An example of doing that is [here](https://github.com/facebook/react-native/blob/main/.circleci/config.yml#L790)

Updated, thank you